libs/community/langchain_community/embeddings/deepinfra.py

```diff
@@ -6,6 +6,7 @@
 from langchain_core.utils import get_from_dict_or_env
 
 DEFAULT_MODEL_ID = "sentence-transformers/clip-ViT-B-32"
+MAX_BATCH_SIZE = 1024
```

Comment:
Could you expose this as part of the initializer with a default value? Presumably this can vary over time or folks may want to optimize it in a different way.

Embeddings is not the most obvious place to control the batch size. Usually embedding is tied to an indexing process which will be associated with a smaller batch size.

Reply:
Thanks for your feedback @eyurtsev . PR was updated as desired.